### ISSUE TITLE
Installation Manager

### ISSUE THREAD

**Issue by Antel37** [platformio/platformio-core-installer#2011]
%23 Description of problem
  Leave a comment...

  BEFORE SUBMITTING, PLEASE SEARCH FOR DUPLICATES IN
  - https://github.com/platformio/platformio-vscode-ide/issues%3Fq=is%3Aissue 

  %23 Configuration

  VSCode: 1.65.2
  PIO IDE: v2.2.0
  System: Windows_NT, 10.0.19043, x64

  %23 Exception
  ```
  Error: Traceback (most recent call last):
  File "C:\Users\antel\.platformio\.cache\tmp\get-platformio-0.3.5.py", line 69, in <module>
    main()
  File "C:\Users\antel\.platformio\.cache\tmp\get-platformio-0.3.5.py", line 61, in main
    bootstrap()
  File "C:\Users\antel\.platformio\.cache\tmp\get-platformio-0.3.5.py", line 47, in bootstrap
    pioinstaller.__main__.main()
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\pioinstaller\__main__.py", line 122, in main
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\click\core.py", line 829, in __call__
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\click\core.py", line 782, in main
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\click\core.py", line 1236, in invoke
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\click\core.py", line 1066, in invoke
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\click\core.py", line 610, in invoke
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\click\decorators.py", line 21, in new_func
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\pioinstaller\__main__.py", line 52, in cli
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\pioinstaller\core.py", line 72, in install_platformio_core
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\pioinstaller\penv.py", line 52, in create_core_penv
  File "C:\Users\antel\.platformio\.cache\tmp\.piocore-installer-t3ojn53c\tmpphpf7moc\pioinstaller.zip\pioinstaller\python.py", line 170, in find_compatible_pythons
  File "C:\Users\antel\.platformio\python3\lib\subprocess.py", line 424, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "C:\Users\antel\.platformio\python3\lib\subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Users\antel\.platformio\python3\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\antel\.platformio\python3\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
OSError: [WinError 1] Funci�n incorrecta
 -> Installer version: 0.3.5
Platform: Windows-10
Python version: 3.9.10 (tags/v3.9.10:f2f3f53, Jan 17 2022, 15:14:21) [MSC v.1929 64 bit (AMD64)]
Python path: C:\Users\antel\.platformio\python3\python.exe
Creating a virtual environment at C:\Users\antel\.platformio\penv

	at c:\Users\antel\.vscode\extensions\platformio.platformio-ide-2.2.0\node_modules\platformio-node-helpers\dist\index.js:1:879275
	at ChildProcess.c (c:\Users\antel\.vscode\extensions\platformio.platformio-ide-2.2.0\node_modules\platformio-node-helpers\dist\index.js:1:879169)
	at ChildProcess.emit (events.js:315:20)
	at ChildProcess.cp.emit (c:\Users\antel\.vscode\extensions\platformio.platformio-ide-2.2.0\node_modules\cross-spawn\lib\enoent.js:34:29)
	at maybeClose (internal/child_process.js:1048:16)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
  ```

**Comment by Antel37** [comment]
SOLVED:
After consulting in all possible forums, without finding a solution, I found that on another computer, which I use very little, "platformio" worked correctly. So I opted for:
1.- Delete the folder "C:\Users\user\.platformio" that did not work.
2.- Paste the ".platformio" folder that worked in its place.
3.- And solved "Platformio" WORKS CORRECTLY.
Since then I keep a ".platformio" folder, preventively.
(I'm posting it in case someone finds it useful.)
